src/schema/uuid/thread/resolvers.ts

```diff
@@ -20,31 +20,32 @@
  * @link      https://github.com/serlo-org/api.serlo.org for the canonical source repository
  */
 import { ApolloError } from 'apollo-server'
-import * as R from 'ramda'
 
 import { resolveConnection } from '../../connection'
 import { createUuidResolvers, UuidPayload } from '../abstract-uuid'
 import { UserPayload } from '../user'
 import { CommentPayload, ThreadDataType, ThreadResolvers } from './types'
-import { assertUserIsAuthenticated } from '~/schema/utils'
+import { decodeThreadId, encodeThreadId } from './utils'
+import {
+  assertUserIsAuthenticated,
+  createMutationNamespace,
+} from '~/schema/utils'
 
+//TODO: add filter for trashed threads and comments
 export const resolvers: ThreadResolvers = {
   Thread: {
-    createdAt(thread, _args) {
-      return thread.commentPayloads.map((comment) => comment.date).reduce(R.min)
+    id(thread, _args) {
+      return encodeThreadId(thread.commentPayloads[0].id)
     },
-    updatedAt(thread, _args) {
-      return thread.commentPayloads.map((comment) => comment.date).reduce(R.max)
+    createdAt(thread, _args) {
+      return thread.commentPayloads[0].date
     },
     title(thread, _args) {
```

Comment:
```suggestion
    title(thread) {
```

in other places as well...